### PR TITLE
issue-14868-revert-redirect-site-change

### PR DESCRIPTION
**Pull request by alfredo-dotcms**

### CODE REVIEW

File: src/app/view/components/dot-toolbar/dot-toolbar.component.ts [hunk 1]
```diff
@@ -33,7 +35,9 @@ export class ToolbarComponent implements OnInit {
 
     siteChange(site: Site): void {
         this.siteService.switchSite(site);
-        this.dotNavigationService.goToFirstPortlet();
+        if (this.route.snapshot['_routerState'].url.indexOf('edit-page/') >= 0) {
+            this.dotNavigationService.goToFirstPortlet();
```
Comment (fmontes) [inline src/app/view/components/dot-toolbar/dot-toolbar.component.ts]: What we talked is that it should be to `site-browser` and if anything happen, lilke the user doesn't have access to the site browser the router will redirect to first portlet automatically.

File: src/app/view/components/dot-toolbar/dot-toolbar.component.ts [hunk 2]
```diff
@@ -33,7 +35,9 @@ export class ToolbarComponent implements OnInit {
 
     siteChange(site: Site): void {
```
Comment (fmontes) [inline src/app/view/components/dot-toolbar/dot-toolbar.component.ts]: So if I'm in "site browser" and I'm seeing like 20 folders and I change the site, do the site browser update to reflect the folders for the new site? (the same goes with all the portlets)

Comment (alfredo-dotcms) [reply src/app/view/components/dot-toolbar/dot-toolbar.component.ts]: yes it is updated

File: src/app/view/components/dot-toolbar/dot-toolbar.component.ts [hunk 3]
```diff
@@ -33,7 +33,9 @@ export class ToolbarComponent implements OnInit {
...
         this.siteService.switchSite(site);
-        this.dotNavigationService.goToFirstPortlet();
+        if (this.route.snapshot['_routerState'].url.indexOf('edit-page/') >= 0) {
```
Comment (fmontes) [inline src/app/view/components/dot-toolbar/dot-toolbar.component.ts]: I think we should create a method in the `dotRouterService` like `isEditPage()` to do this. @freddyucv @hmoreras?

Comment (hmoreras) [reply src/app/view/components/dot-toolbar/dot-toolbar.component.ts]: sounds like it will be reusable. agree.

**Pull request merged**